src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectVotingController.kt

```diff
@@ -0,0 +1,91 @@
+package com.terraformation.backend.accelerator.api
+
+import com.terraformation.backend.api.AcceleratorEndpoint
+import com.terraformation.backend.api.ApiResponse200
+import com.terraformation.backend.api.ApiResponse403
+import com.terraformation.backend.api.ApiResponse404
+import com.terraformation.backend.api.SuccessResponsePayload
+import com.terraformation.backend.db.accelerator.CohortPhase
+import com.terraformation.backend.db.accelerator.VoteOption
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.UserId
+import io.swagger.v3.oas.annotations.Operation
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@AcceleratorEndpoint
+@RequestMapping("/api/v1/accelerator/voting/")
+@RestController
+class ProjectVotingController() {
+  @ApiResponse200
+  @ApiResponse403
+  @ApiResponse404
+  @GetMapping("/{projectId}")
```

Comment:
Should we have another layer in the path here? Like `/projects/{projectId}` or maybe flip it around so it's `/api/v1/accelerator/projects/{projectId}/voting`?

I'm thinking that at some point we'll have a voting-related endpoint that doesn't target a single project and we'd want something other than a project ID at this path position.

Reply:
I think this makes more sense to me. I think from the design mocks, it does point towards project being a center piece and functionalities built on top of it